index.html

```diff
@@ -41,18 +41,21 @@ <h3>Three Favorite Video Games</h3>
                     <li>Super Mario Strikers Charged</li>
                 </ol>
                 <h3>Haiku</h3>
-                    <p>Autumn moonlight-</p>
-                    <p>a worm digs silently</p>
-                    <p>into the chestnut.</p>
-                <cite> by <span class="author">Matsuo Basho</span></cite>
+                <p>
+                    Autumn moonlight-<br>
+                    a worm digs silently<br>
+                    into the chestnut.<br>
+                    <cite> by <span class="author">Matsuo Basho</span></cite>
+                </p>
```

Comment:
Each line isn't a separate paragraph, so use `<br>` to note new lines rather than `<p>`